tools/qr_coder/tool_test_output.json

```diff
@@ -0,0 +1 @@
+{"tests": [{"data": {"status": "success", "inputs": {"qrData": "http://upmc.fr"}, "job": {"inputs": {}, "update_time": "2015-09-23T08:18:11.958266", "tool_id": "qr_coder_1", "outputs": {"qrOutput": {"src": "hda", "id": "2891970512fa2d5a", "uuid": "cb5f6d98-c1d4-430c-a328-0ab1817be601"}}, "stdout": "#Parameters passed by user\nqrErrorCorrection [Low]\nqrFit [True]\nqrBoxSize []\nqrOutput [/tmp/tmpKqacMf/files/000/dataset_1.dat]\nqrVersion []\nqrData [http://upmc.fr]\nqrBorder []\n\n", "command_line": "python /home/fabiorjvieira/Cloud/GitHub/tools-artbio/tools/qr_coder/qr_coder.py -qrData \"http://upmc.fr\" -qrOutput /tmp/tmpKqacMf/files/000/dataset_1.dat --qrFit -qrErrorCorrection Low", "exit_code": 0, "state": "ok", "create_time": "2015-09-23T08:18:04.044349", "params": {"qrErrorCorrection": "\"Low\"", "qrFit": "\"True\"", "qrBoxSize": "\"\"", "dbkey": "\"?\"", "qrVersion": "\"\"", "qrData": "\"http://upmc.fr\"", "chromInfo": "\"/home/fabiorjvieira/clean_galaxy/galaxy/tool-data/shared/ucsc/chrom/?.len\"", "qrBorder": "\"\""}, "stderr": "", "job_metrics": [], "model_class": "Job", "external_id": "19486", "id": "2891970512fa2d5a", "user_email": "test@bx.psu.edu"}}, "id": "functional.test_toolbox.TestForTool_qr_coder_1.test_tool_000000", "has_data": true}], "version": "0.1", "summary": {"num_skips": 0, "num_errors": 0, "num_failures": 0, "num_tests": 1}}
```

Comment:
The test output should not be in the repository.
Perhaps put tool_test_output.json and tool_test_output.html in your .gitignore?